### PR TITLE
Quick polish of Issue Templates

### PR DESCRIPTION
**Pull request by paolaperaza**
Had to 🙃 Two things:

- Capitalized titles where I could just to make things feel tidier
- Actually removed the `New Documentation` request. It seems like people just use the top one we have, and the fields we require aren't all that different

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -26,17 +27,26 @@ body:
...
-      description: Describe the issue and what you think needs to be added, updated, or otherwise changed. Be as descriptive as possible.
-      placeholder:  
+      label: The Issue or Context
+      description: Describe what exactly is wrong or missing in the context of current documentation. Be as descriptive as possible.
+    validations:
+      required: true
+  - type: textarea
+    id: fix
+    attributes:
+      label: The Fix
```
Comment (paolaperaza) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:39]:
```suggestion
      label: Fix or Proposed Change
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -1,21 +1,22 @@
-name: Report an issue in documentation
-description: Any request related to existing documentation that has an issue
-labels: ["error/wrong-info", "error/misleading"]
+name: General Docs Request
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:1]:
```suggestion
name: Request a Docs Change
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
-        - Inaccurate content
-        - Missing or misleading content
-        - Typo
-        - Bug/ broken link
+        - Inaccurate, misleading, or out-of-date information
+        - A new set of best practices or recommendations
+        - A new technical overview, explanation, or reference
+        - Missing content for an existing feature
+        - Typo or broken link
+        - Issue with Docs website
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:19]:
```suggestion
        - Inaccurate, misleading, or out-of-date information
        - Missing content for an existing feature
        - Typo or broken link
        - Issue with Docs website functionality
        - Request a new set of best practices or recommendations
        - Request a new technical overview, explanation, or reference
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 4]
```diff
@@ -26,17 +27,26 @@ body:
     validations:
       required: true
   - type: textarea
-    id: description
+    id: issue
     attributes:
-      label: Describe the Issue and Proposed Fix
-      description: Describe the issue and what you think needs to be added, updated, or otherwise changed. Be as descriptive as possible.
-      placeholder:  
+      label: The Issue or Context
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:32]:
```suggestion
      label: The issue or context
```
I'd prefer not to capitalize the fields themselves

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
 body:
   - type: markdown
     attributes:
       value: |
-        If you want to submit a quick fix yourself, you don't need to write an issue. See [Submit a docs PR](https://www.notion.so/astronomerio/Submit-a-Docs-PR-a3ca9018fdbc4206b120a6d530a8401b).
+        If you want to contribute a fix yourself, you don't need to write an issue. See [Submit a Docs PR](https://www.notion.so/astronomerio/Submit-a-Docs-PR-a3ca9018fdbc4206b120a6d530a8401b).
   - type: dropdown
     id: issuetype
     attributes:
       label: Issue Type
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:11]:
```suggestion
      label: Issue type
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 1 above)
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:39]:
```suggestion
      label: Fix or proposed change
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 6]
```diff
@@ -26,17 +27,26 @@ body:
...
       required: true
   - type: textarea
-    id: description
+    id: issue
     attributes:
-      label: Describe the Issue and Proposed Fix
-      description: Describe the issue and what you think needs to be added, updated, or otherwise changed. Be as descriptive as possible.
-      placeholder:  
+      label: The Issue or Context
+      description: Describe what exactly is wrong or missing in the context of current documentation. Be as descriptive as possible.
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:33]:
```suggestion
      description: Describe what is wrong or missing in the documentation. Be as descriptive as possible.
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 7]
```diff
@@ -26,17 +27,26 @@ body:
...
     id: notes
     attributes:
       label: Additional Notes
       description: Add any related Slack threads, Zendesk tickets, or other information that the docs team can use to resolve the issue.
-
-  
+  - type: input
+    id: reviewers
+    attributes:
+      label: Required Reviewers
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:51]:
```suggestion
      label: Required reviewers
```

File: .github/ISSUE_TEMPLATE/new-feature.yaml [hunk 8]
```diff
@@ -28,12 +28,10 @@ body:
   - type: textarea
     id: deadline
     attributes:
-      label: Release date
+      label: Release Date
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/new-feature.yaml:31]:
```suggestion
      label: Release date
```
Looking back on the templates I'd prefer to uncapitalize these topics to match our docs site style, but don't feel strongly.

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 9]
```diff
@@ -1,21 +1,22 @@
-name: Report an issue in documentation
-description: Any request related to existing documentation that has an issue
-labels: ["error/wrong-info", "error/misleading"]
+name: General Docs Request
+description: Anything in existing documentation that is wrong, missing, or misleading.
```
Comment (jwitz) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:2]:
```suggestion
description: Use this template to request changes to existing Astronomer docs, or to request new content.
```

**Review by jwitz — CHANGES_REQUESTED** [review]
This is an interesting design @paolaperaza. While I'm all for simplifying templates and choice, I think we've inadvertently created some new complexities that we'll have to resolve before this is merged. See my comments for more details

**Review by paolaperaza — COMMENTED** [review]
@jwitz Back to you!

**Pull request merged**